docs/guided-remediation.md

```diff
@@ -246,6 +246,10 @@ If your project uses mirrored or private registries, you will need to use `--dat
 ## Known issues
 
 - The subcommand does not use the `osv-scanner.toml` configuration. Use the `--ignore-vulns` flag instead.
+- The subcommand does not group aliases of the same vulnerabilities together.
+- Unique vulnerabilities are counted differently when relocking & relaxing versus with in-place and the `scan` action. Scan will count the same OSV ID affecting two different package versions separately, whereas relock will count this as one vulnerability.
```

Comment:
This is a little hard to parse. 

Can we replace references to relocking with `fix --strategy=relock` and references to in-place with `fix --strategy=in-place` ? And references to the scan command with `scan` ?

Reply:
Replaced with subcommands & flags.